oqs_scripts/testrun.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+if [ ! -f build/tool/bssl ]; then
+   echo "Test executable not available at 'build/tool/bssl'. Exiting."
+   exit 1
+fi
+
+rm -rf testrun && mkdir testrun && cd testrun
+
+# pull current CA cert
+wget https://test.openquantumsafe.org/CA.crt
+
+# pull list of algs/ports
+wget https://test.openquantumsafe.org/assignments.json
```

Comment:
Can we just retrieve `CA.crt` and `assignments.json` in the python test script and do away with this bash script all together?

Reply:
Agreed.